src/components/Article.jsx

```diff
@@ -1 +1,35 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => {
```

Comment:
```suggestion
export const Article = ({ 
  title, 
  date, 
  text 
}) => {
```

if we have three or more values, better to start them from new line